src/Stripe.Tests/transfers/when_creating_a_transfer_with_application_fee.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ***** This test needs a bank account setup in order to run correctly, and the ignore attribute wasn't working. Good times.
+
+//using Machine.Specifications;
+
+//namespace Stripe.Tests
+//{
+//    public class when_creating_a_transfer_with_application_fee
+//    {
+//        protected static StripeTransferCreateOptions StripeTransferCreateOptions;
+//        protected static StripeTransfer StripeTransfer;
```

Comment:
This can be private static - it needs to be global because it's used in the assert.